power_grid_model_c/power_grid_model/include/power_grid_model/main_core/core_utils.hpp

```diff
@@ -12,6 +12,8 @@
 
 namespace power_grid_model::main_core::utils {
 
+constexpr Idx invalid_index_v{-1};
```

Comment:
What does `_v` imply?

Reply:
But you are not trying to extract `::value` from a template argument, right? That's the more appropriate context.

Reply:
You are right. I amended this in [c129a50](https://github.com/PowerGridModel/power-grid-model/pull/843/commits/c129a505abde61f497717f07ba7a2114328f1f09).